chapter_computational-performance/hardware_vn.md

```diff
@@ -177,7 +177,20 @@ We can safely ignore the details for the purpose of this book.
 They only matter when tuning GPU kernels for high throughput.
 -->
 
-*dịch đoạn phía trên*
+Bộ nhớ GPU ràng buộc đối với các yêu cầu cần băng thông thậm chí cao hơn vì chúng có nhiều phần tử xử lý hơn CPU.
+Nhìn chung có hai lựa chọn để định địa chỉ chúng. 
```

Comment:
```suggestion
Nhìn chung có hai phương án để đối mặt với vấn đề này. 
```

Reply:
Ở đây đang nói về bộ nhớ nên từ addressing trong phần cứng có nghĩa là định địa chỉ.

Reply:
Em không hiểu vậy. Đúng là đang bàn về bộ nhớ nhưng không nhắc hề nhắc gì tới việc định địa chỉ ô nhớ. Đây chỉ đang bàn tới 2 cách giải quyết vấn đề cần thêm băng thông: `One is to make the memory bus significantly wider.`, `An alternative is to use HBM (high bandwidth memory) modules.`

Reply:
Mình nghĩ có thể đổi thành "Nhìn chung có hai phương án tiếp cận đối với vấn đề này"

Reply:
```suggestion
Nhìn chung có hai phương án tiếp cận đối với vấn đề này.
```